assets/all.settings.php

```diff
@@ -60,6 +69,16 @@
   $config['openid_connect.settings.adgangsplatformen']['settings']['client_id'] = 'dummy-id';
   $config['openid_connect.settings.adgangsplatformen']['settings']['client_id'] = 'dummy-secret';
   $config['openid_connect.settings.adgangsplatformen']['settings']['agency_id'] = '100200';
+
+  // Set service base urls for the react apps.
+  // We need http domains for testing in CI context.
+  $config['dpl_react_apps.services'] = [
+    'fbs' => ['base_url' => 'http://fbs-openplatform.dbc.dk'],
```

Comment:
Please reconsider this definition.

The current value duplicates what is already available for `$config['dpl_fbs.settings']`. We only need one of them.

The current approach was chosen due to an expectation of a future `dpl_fbs` module as hinted danskernesdigitalebibliotek/dpl-cms#131 and danskernesdigitalebibliotek/dpl-cms#153 and repeated in danskernesdigitalebibliotek/dpl-cms#186. The current approach goes against this by centralizing all service urls in the `dpl_react_apps` module. This is bound to cause conflicts between the two approaches.

Naming might also cause confusion. Configuration is generally in the format `[module_name].settings.foo.bar`. This uses `[module_name].services`. Services has a different meaning in Drupal/Symfony.

Reply:
The intention for the structure of the FBS url was that it would belong to a future `dpl_fbs` module.  danskernesdigitalebibliotek/dpl-cms#186 intends to add this and a `dpl_publizon` module. This points to a module per service approach.

If we are to follow this approach then all the remaining urls could be places in similar configuration e.g. `dpl_cover_service`, `dpl_fbi, `dpl_material_list` etc. The current approach shows that there does not have to be a module for the configuration to exist. On the other hand one might argue that it is silly to use configuration names for something that does not existing and we have no plan to create in the foreseeable future.

In any case: The code as is in direct conflict with what already exists for FBS and [what is being implemented for Publizon](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/186/files#diff-ed2ad4e359fcd058de99be911d88afeaf7a88e96d6a9b45d9974d4976747863a). I suggest that we align the configuration of these two services with their corresponding modules.

For the rest of the services I am personally fine with either keeping them as is or "preparing" for future service-specific modules.

Reply:
I'l l fix